docs/playbooks/wind-river-cloud-platform-deployment-manager.yaml

```diff
@@ -744,6 +744,7 @@
               timeout: "{{ boot_wait_time }}"
             register: waiting_after_reboot
             ignore_unreachable: true
+            ignore_errors: true
```

Comment:
About the cases "ignore_unreachable" and "ignore_errors" for future reference-

Unreachable:
It can ignore a task failure due to the host instance being ’UNREACHABLE’ with the ignore_unreachable keyword. Ansible ignores the task errors but continues to execute future tasks against the unreachable host.
This usually happens if the host is network-inaccessible. For example, during a second reboot that temporarily disrupts SSH connectivity, Ansible will mark it as unreachable since it can’t connect to the host at all.

Failed: It can ignore errors when the task is failed, except those syntax errors, undefined variable errors, connection failures, execution issues.
This occurs if Ansible can reach the host, but the task itself encounters an issue. This could happen if the host is reachable but not fully ready to respond, or if a required port isn’t open yet.